### ISSUE TITLE
Some buttons in YouTube embeds don't work

### ISSUE THREAD

**Issue by sdtransier** [brentsimmons/NetNewsWire#553]
Version 5.0d15 (1517)

If there is a YouTube embed in the feed, the following buttons/links don't work:

- Main title
- Watch Later
- "YouTube" button on bottom bar
- Fullscreen

I don't know if this is a YouTube embed limitation or NNW issue. 

<img width="659" alt="screen shot 2019-02-21 at 7 50 15 am" src="https://user-images.githubusercontent.com/17955542/53177538-78929f00-35ad-11e9-8209-655b3a337750.png">

**Comment by brentsimmons** [comment]
What’s the URL of the feed where you’re seeing this?

**Comment by sdtransier** [comment]
It's the most recent post in http://feeds.kottke.org/json

**Comment by brentsimmons** [comment]
I tested — none of my code in NetNewsWire gets called on clicks on any of these things.

**Comment by joshuajabbour** [comment]
Yes, I’m noticing the same issue. Always happens on kottke.org YouTube embeds, though I have no issues on the videos when viewed directly on his site. I found a video embedded in another NNW default feed (the iBeer post on furbo.org), which also exhibits the issue. 

I just spent a while debugging this further, and was able to reproduce in other feed readers. News explorer displays the embeds inline like NNW, and also has the issue. Neither Reeder 3 nor Unread 1 display embedded videos inline, so they don’t have the issue. 

Therefore I assume it’s an issue with YouTube’s embed code in some way. Unfortunate, because it’s super annoying to not be able to open videos directly in the YouTube app.

**Comment by jimtut** [comment]
Not sure I agree w/ the last comment.  My feeds include MacRumors and Dark Horizons, and I've used the old NNW (3 or 4?) on a very old Mac without problems w/ this.  Upgrading to Big Sur and NNW 5 with the same feeds gives me this problem.

I've tried w/ the same feeds saved "locally" and with them stored in Feedly.  Definitely a problem w/ the Mac desktop version, haven't tried much w/ the iOS version, but I think it's the same.

**Comment by brentsimmons** [comment]
If those worked in old NetNewsWire versions, it’s almost certainly because we were using the old WebView instead of the new WKWebView, which replaces it. They’re similar, yes, but different enough to cause a variety of issues.

**Comment by brentsimmons** [comment]
Can't reproduce with the 7.1.3 development version. Closing this. We can reopen after 7.1.3 ships if it’s still an issue.
